templates/includes/navbar_user.html.twig

```diff
@@ -8,18 +8,19 @@
 {% endif %}
 {% if user is not null %}
     <div class="nav-item dropdown">
-        <a href="#" class="nav-link d-flex lh-1 text-reset p-0" data-bs-toggle="dropdown" aria-label="{{ 'Open personal menu'|trans({}, 'TablerBundle') }}">
+        <a href="#" class="nav-link d-flex lh-1 p-0 px-2" data-bs-toggle="dropdown" aria-label="{{ 'Open personal menu'|trans({}, 'TablerBundle') }}">
             {% import "@Tabler/components/avatar.html.twig" as macro %}
-            {{ macro.avatar(user, 'sm') }}
+            {{ macro.avatar({imageUrl : asset(user.avatar), size : 'sm'}) }}
```

Comment:
The asset call is unexpected, why do we have that?
What if user.avatar is an absolute URL?